migration/table.go

```diff
@@ -0,0 +1,250 @@
+package migration
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+	"strings"
+
+	"github.com/cloudquery/cq-provider-sdk/migration/longestcommon"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+	"github.com/georgysavva/scany/pgxscan"
+	"github.com/hashicorp/go-hclog"
+	"github.com/jackc/pgx/v4/pgxpool"
+	"github.com/thoas/go-funk"
+)
+
+const (
+	queryTableColumns   = `SELECT array_agg(column_name::text) AS columns, array_agg(data_type::text) AS types FROM information_schema.columns WHERE table_name = $1 AND table_schema = $2`
+	addColumnToTable    = `ALTER TABLE %s ADD COLUMN IF NOT EXISTS %v %v`
+	dropColumnFromTable = `ALTER TABLE %s DROP COLUMN IF EXISTS %v`
```

Comment:
add if exists to these migrations?, shouldn't they also end with `;` ?

Reply:
Added. They didn't, but since they look weird I now added the `;` and update code.